npm/ls-lint/bin/index.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+var spawn = require('child_process').spawn;
+var path = require('path');
+
+var command_args = process.argv.slice(2);
+
+function spawnCommand(binaryExecutable) {
+  var child = spawn(
+    path.join(__dirname, binaryExecutable),
+    command_args,
+    { stdio: [process.stdin, process.stdout, process.stderr] }
+  );
+
+  child.on('close', function(code) {
+    if (code !== 0) {
+      process.exit(1);
+    }
+  });
+}
+
+if (process.platform === 'darwin') {
```

Comment:
please use switch case here, cause there could be some multi cases like here:

https://github.com/electron/electron/blob/master/npm/install.js#L67